database/seeders/ConfigurationSeeder.php

```diff
@@ -19,52 +19,66 @@ public function run()
             'key' => 'INITIAL_CREDITS',
         ], [
             'value' => '250',
-            'type'  => 'integer',
+            'type' => 'integer',
         ]);
 
         Configuration::firstOrCreate([
             'key' => 'INITIAL_SERVER_LIMIT',
         ], [
             'value' => '1',
-            'type'  => 'integer',
+            'type' => 'integer',
         ]);
 
         //verify email event
         Configuration::firstOrCreate([
             'key' => 'CREDITS_REWARD_AFTER_VERIFY_EMAIL',
         ], [
             'value' => '250',
-            'type'  => 'integer',
+            'type' => 'integer',
+        ]);
+
+        Configuration::firstOrCreate([
+            'key' => 'VERIFIED_EMAIL_TO_MAKE_SERVER',
+        ], [
+            'value' => 'true',
+            'type' => 'boolean',
+        ]);
+
+        Configuration::firstOrCreate([
+            'key' => 'VERIFIED_DISCORD_TO_MAKE_SERVER',
+        ], [
+            'value' => 'true',
```

Comment:
not much besides turning on the extra spacing options for array's :P 
could you also rename the keys to something like (FORCE_EMAIL_VERIFICATION)
and perhaps add a small description explaining what this option does :)
I've already done this to all of the config options, but it seems like you are running an older version.

Reply:
Done in new PR (#95) (sadly I was not able to commit into this one)